src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -34,12 +39,15 @@ const ImageWithPlaceholder = ({
   width,
 }) => {
   const { isAmp } = useContext(RequestContext);
-
+  const [isLoaded, setIsLoaded] = useState(false);
+  const ImageParent = isLoaded ? ImageWithoutPlaceholder : ImagePlaceholder;
   const imageProps = { alt, src, sizes, width, srcset, fade };
-  const imageToRender = <Image {...imageProps} />;
+  const imageToRender = (
+    <Image onLoad={() => setIsLoaded(true)} {...imageProps} />
+  );
 
   return (
-    <ImagePlaceholder ratio={ratio}>
+    <ImageParent ratio={ratio}>
```

Comment:
## Other Solutions Assessed
**1. Using Styled Components CSS Prop**
```js
<ImagePlaceholder css={hidePlaceholder && `background: none`}>
...
</ImagePlaceholder>
```

Nice and tidy. Though it causes a lot of snapshot updates, due to Styled Components internal workings (I assume). Not a bad thing.

**2. Nesting `ImagePlaceholder` inside another SC that has some background logic**
```js
const StyledPlaceholder = styled(ImagePlaceholder)`
  background: ${({ isLoaded }) => isLoaded && 'none'};
`;
```

Adds an extra class as above, therefore same issue with snapshots.


This solution doesn't make any changes to snapshots, and still works as expected.